pom.xml

```diff
@@ -156,6 +156,11 @@
     </dependency>
     <dependency>
       <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
```

Comment:
I have no clue. I know that [IteratorIterable](https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/iterators/IteratorIterable.html) is in colletions4, and it didn't seem to compile without this.